### PR TITLE
Login to comment

### PR DESCRIPTION
**Pull request by mariom3**
Addresses a few comments from PR #14:
- One servlet for comments and another for logging in/out.
- Separate JS functions pull from each of the servlets 
- Logout link works
- Live server updated.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/Comments.java [hunk 1]
```diff
@@ -29,10 +29,9 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+@WebServlet("/comments")
+public class Comments extends HttpServlet {
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/Comments.java]: Maybe call it CommentServelet to be consistent

File: portfolio/src/main/java/com/google/sps/servlets/Comments.java [hunk 2]
```diff
@@ -29,10 +29,9 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+@WebServlet("/comments")
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/Comments.java]: Add a java doc for the class. A comment describing what the class does
/** Servelet for handling comments posted by the user. */

File: portfolio/src/main/java/com/google/sps/servlets/LoginLogout.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/login-logout")
+public class LoginLogout extends HttpServlet {
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/LoginLogout.java]: Call it LoginLogoutSeverlet and add javadoc

**Review by sbahaud — COMMENTED** [review]
Left some comments

File: portfolio/src/main/java/com/google/sps/servlets/LoginLogout.java (hunk 3 above)
Comment (mariom3) [reply portfolio/src/main/java/com/google/sps/servlets/LoginLogout.java]: Done: 29fdbd4

File: portfolio/src/main/java/com/google/sps/servlets/Comments.java (hunk 2 above)
Comment (mariom3) [reply portfolio/src/main/java/com/google/sps/servlets/Comments.java]: Done: 29fdbd4

File: portfolio/src/main/java/com/google/sps/servlets/Comments.java (hunk 1 above)
Comment (mariom3) [reply portfolio/src/main/java/com/google/sps/servlets/Comments.java]: Done: 29fdbd4

**Review by sbahaud — APPROVED** [review]
LGTM

**Pull request merged**